topic_message_query.go

```diff
@@ -27,11 +27,13 @@ type TopicMessageQuery struct {
 }
```

Comment:
There's a bug in this file. It should be assigning body.ConsensusEndTime:
```go
	if query.endTime != nil {
		body.ConsensusStartTime = _TimeToProtobuf(*query.endTime)
	}
```

Reply:
Thanks for catching this